src/gpl/src/initialPlace.h

```diff
@@ -102,6 +106,9 @@ class InitialPlace {
     Eigen::VectorXf instLocVecX_, fixedInstForceVecX_;
     Eigen::VectorXf instLocVecY_, fixedInstForceVecY_;
     SMatrix placeInstForceMatrixX_, placeInstForceMatrixY_;
+    bool GPU = 1;
```

Comment:
Style nit: in C++ we generally prefer "true" / "false" keywords to 1/0 when the value is boolean, and avoid explicit comparisons to `== 1` preferring just `if (myBoolean)` and such.